multimodal/tests/unittests/test_distiller.py

```diff
@@ -81,8 +98,9 @@ def test_distillation():
     predictor = predictor.fit(
         train_data=dataset.train_df,
         teacher_predictor=teacher_predictor.path,
-        hyperparameters=hyperparameters,
+        hyperparameters=student_hyperparameters,
         time_limit=30,
         save_path=student_save_path,
     )
-    verify_predictor_save_load(predictor, dataset.test_df)
+
+    verify_predictor_save_load(predictor, dataset.test_df)
```

Comment:
new line missing at the end of this file.

Reply:
done